src/adapter/graphql/tests/tests/test_gql_metadata_chain.rs

```diff
@@ -128,7 +128,7 @@ async fn test_metadata_chain_events() {
         .await;
     assert!(res.is_ok(), "{res:?}");
 
-    let expected_schema = r#"{"name": "arrow_schema", "type": "struct", "fields": [{"name": "city", "repetition": "REQUIRED", "type": "BYTE_ARRAY", "logicalType": "STRING"}, {"name": "population", "repetition": "REQUIRED", "type": "INT64"}]}"#;
+    let expected_schema = r#"{"fields":[{"name":"city","data_type":"Utf8","nullable":false,"dict_id":0,"dict_is_ordered":false,"metadata":{}},{"name":"population","data_type":"Int64","nullable":false,"dict_id":0,"dict_is_ordered":false,"metadata":{}}],"metadata":{}}"#;
```

Comment:
Let's improve this test to also use `json!({ })` macro. It's very hard to read and fragile.

Reply:
Actually you might be able to use `value!()` macro in the same way as assertion below constructs it to compare the whole body of the response.

Reply:
done